config/certsuite_config.yml

```diff
@@ -38,3 +38,8 @@ executedBy: ""
 partnerName: ""
 collectorAppPassword: ""
 collectorAppEndpoint: "http://claims-collector.cnf-certifications.sysdeseng.com"
+connectAPIKey: ""
```

Comment:
Suggestion: make a placeholder field for all these params, with a bit more descriptive name like"redhat-connect-api":
```
redhat-connect-api:
  projectID: ...
  key: ...
  baseURL: ...
  ...
```
or
```
redhat-connect:
  projectID: ...
  api: 
    key: ...
    baseURL: ...
    ...
```

Reply:
Done!